nservicebus/hosting/nservicebus-host/index.md

```diff
@@ -25,6 +25,15 @@ If you want to avoid the scanning process you can explicitly configure the type
 snippet:ExplicitHostConfigType
 
 
+### Controlling assembly scanning using the command line
+
+A list of assemblies to scan can also be controlled using the `/scannedAssemblies` switch. If this option is used, the `NServiceBus.Host.exe` loads only assemblies (with references) that have been explicitly listed on the command line. Each assembly must be added using a separate switch:
+
+`NServiceBus.Host.exe /scannedAssemblies:"NServiceBus.Host" /scannedAssemblies:"MyMessages" /scannedAssemblies:"MyEndpoint"`
+
+NOTE: The `NServiceBus.Host` assembly must be listed explicitly. In this case `NServiceBus.Core` is loaded as a reference and need not be specified separately.
```

Comment:
I fixed the wording. I don't know why this is the case that NServiceBus.Host has to be loaded. I verified this empirically only. Maybe someone familiar with the matter can comment.

/cc @Particular/nservicebus-maintainers